features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/wrappers/BitmapWrapper.kt

```diff
@@ -27,4 +27,25 @@ internal class BitmapWrapper {
             null
         }
     }
+
+    internal fun createScaledBitmap(
```

Comment:
I wonder why do you need the `createBitmap` above ? As that one only delegates to this one.